components/edit-collective/UpdateBankDetailsForm.js

```diff
@@ -40,10 +40,12 @@ class UpdateBankDetailsForm extends React.Component {
   }
 
   onChange(field, value) {
-    const newState = this.state;
-    newState.form[field] = value;
-    this.setState(newState);
-    this.props.onChange(newState.form);
+    this.setState(state => {
+      const newState = { ...state };
+      newState.form[field] = value;
+      this.props.onChange(newState.form);
```

Comment:
Do we really want that? Sounds weird to trigger such action in setState.

Reply:
It's not done yet :slightly_smiling_face: